modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -71,6 +73,16 @@ def submit_power_of_attorney(poa_code)
           )
         end
 
+        def validate_2122a
+          target_veteran
+          validate_json_schema('2122a'.upcase)
```

Comment:
under the hood, the available schema names are upcased. Need it here to find the file.